### PR TITLE
Add Swahili language

### LINKED ISSUE #19193: Swahili language is not available in the back office
#### Describe the bug

Hi,
I've translated into Swahili language the strings of PrestaShop and I've uploaded it
Unfortunately when I try to add it to my installed PrestaShop I receive the following message:
"Sorry this language is not available".

#### Expected behavior

I expect to be able to download the just translated language.

#### Steps to Reproduce

* PrestaShop version: 1.7.6
* PHP version: 7

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add the support for Swahili language
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #19193
| How to test?  | **TEST 1 :** <br>1. BO > International > Localization <br> 2. Add Kenya localization pack<br>3. :tada: The Swahili has been installed :tada: <br>**TEST 2 :** <br>1. BO > International > Localization <br> 2. Add Tanzania localization pack<br>3. :tada: The Swahili has been installed :tada: 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19282)

### CODE REVIEW

File: app/Resources/all_languages.json [hunk 1]
```diff
@@ -576,6 +576,15 @@
     "language_code": "sv-se",
     "locale": "sv-SE"
   },
+  "sw": {
+    "name": "Kiswahili (Swahili)",
+    "iso_code": "sw",
+    "date_format_lite": "n/d/Y",
```
Comment (LouiseBonnard) [inline app/Resources/all_languages.json:582]:
```suggestion
    "date_format_lite": "m/d/Y",
```

File: app/Resources/all_languages.json [hunk 2]
```diff
@@ -576,6 +576,15 @@
...
+    "date_format_full": "n/d/Y H:i:s",
```
Comment (LouiseBonnard) [inline app/Resources/all_languages.json:583]:
```suggestion
    "date_format_full": "m/d/Y H:i:s",
```

**Review by LouiseBonnard — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Comment by SD1982** [comment]
Hi @Progi1984 The Swahili language is correctly added with Tanzania localization pack but not with Kenia localization pack ( see screen-record )
https://drive.google.com/file/d/1ISdTioELouXl1YfLVwrLiSfVzJV97dd4/view?usp=sharing

**Comment by Progi1984** [comment]
@SD1982 Found the problem. The API wasn't up to date (http://api.prestashop.com/localization/17/ke.xml). So you can check it.

**Review by NeOMakinG — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
Hi @Progi1984 It's ok now the swahili language is added with both localization pack ( Tanzania and Kenya . Thanks !

**Pull request merged**